### PR TITLE
fuzz tick math property

### PR DESCRIPTION
**Pull request by hensha256**

### CODE REVIEW

File: .forge-snapshots/TickMathGetTickAtSqrtPrice.snap [hunk 1]
```diff
@@ -1 +1 @@
-196153
\ No newline at end of file
+195023
```
Comment (hensha256) [inline .forge-snapshots/TickMathGetTickAtSqrtPrice.snap:1]: this is a different test it doesn tmake sense lol

File: test/libraries/TickMath.t.sol [hunk 2]
```diff
@@ -205,6 +205,26 @@ contract TickMathTestTest is Test, JavascriptFfi, GasSnapshot {
         }
     }
 
+    function test_fuzz_getTickAtSqrtPrice_getSqrtPriceAtTick_relation(int24 tick) public pure {
```
Comment (hensha256) [inline test/libraries/TickMath.t.sol:208]: this passes with 1 million runs locally

**Review by snreynolds — COMMENTED** [review]
nice i like it!

File: .forge-snapshots/TickMathGetTickAtSqrtPrice.snap (hunk 1 above)
Comment (snreynolds) [reply .forge-snapshots/TickMathGetTickAtSqrtPrice.snap:1]: super odd.. I guess let's see if this becomes flaky? Not sure why this test changed this val

**Review by snreynolds — APPROVED** [review]

**Pull request merged**